docs/source/theory/attacks/differencing.ipynb

```diff
@@ -287,7 +285,7 @@
     "\n",
     "## Distribution of Outcomes\n",
     "To demonstrate what differential privacy does to protect our data,\n",
-    "lets now take off the adversary hat, and put on the student hat.\n",
+    "lets' now take off the adversary hat, and put on the student hat.\n",
```

Comment:
Sorry my mom was an English teacher: "let's". 

If it helps, remember it's a contraction for "let us", and if that substitution doesn't work, no apostrophe.

> This lets the result have higher accuracy.
vs.
> ~~This let us the result have higher accuracy.~~